examples/testapp/src/context/CBWSDKReactContextProvider.tsx

```diff
@@ -74,8 +74,8 @@ export function CBWSDKReactContextProvider({ children }: CBWSDKProviderProps) {
   useEffect(() => {
     let cbwsdk;
     let preference;
-    if (version === 'HEAD' || version === '4.0.4') {
-      const SDK = version === 'HEAD' ? CoinbaseWalletSDKHEAD : CoinbaseWalletSDK404;
+    if (version === 'HEAD' || version === 'Latest') {
+      const SDK = version === 'HEAD' ? CoinbaseWalletSDKHEAD : CoinbaseWalletSDKLatest;
```

Comment:
Is `CoinbaseWalletSDKLatest` most recent release and `HEAD` the most recent commit?